src/test/java/org/broadinstitute/consent/http/service/DatasetServiceTest.java

```diff
@@ -519,12 +518,13 @@ void testEnforceDAARestrictions() {
   }
 
   @Test
-  void testUpdateDatasetIndex() throws SQLException {
+  void testUpdateIfIndexed() throws SQLException {
+    Dataset dataset = new Dataset();
+    dataset.setIndexedDate(new Date());
     doNothing().when(datasetServiceDAO).updateDatasetIndex(any(), any(), any());
```

Comment:
This is the default behavior for a mock, so it should be unnecessary. If you want to verify that a mock is used, I would use `verify()` rather than relying on strict stubbing (if that's what you're doing here).
```
    verify(datasetServiceDAO, times(2)).updateDatasetIndex(any(), any(), any());
```